test/uk/gov/hmrc/datecalculator/testsupport/stubs/FakeApplicationProvider.scala

```diff
@@ -19,12 +19,16 @@ package uk.gov.hmrc.datecalculator.testsupport.stubs
 import org.scalatestplus.play.guice.GuiceFakeApplicationFactory
 import play.api.Application
 import play.api.inject.guice.{GuiceApplicationBuilder, GuiceableModule}
-import uk.gov.hmrc.http.test.WireMockSupport
+import uk.gov.hmrc.http.test.ExternalWireMockSupport
 
 import java.time.format.DateTimeFormatter
 import java.time.{Clock, LocalTime, ZoneId}
 
-trait FakeApplicationProvider { this: GuiceFakeApplicationFactory with WireMockSupport =>
+trait FakeApplicationProvider { this: GuiceFakeApplicationFactory with ExternalWireMockSupport =>
```

Comment:
following the guidance [here](https://github.com/hmrc/http-verbs#testing) with `ExternalWireMockSupport`. The call to GDS API is external to the platform, so we can test the header propagation rules don't somehow erase the `From` header (see [here](https://github.com/hmrc/http-verbs#propagation-of-headers) as well)